blackbox/docs/admin/cluster-management.rst

```diff
@@ -0,0 +1,101 @@
+.. _cluster_management:
+
+==================
+Cluster management
+==================
+
+With the ``crate-node`` CLI tool you can manage your CrateDB clusters by forming
```

Comment:
Please reword this so that it mirrors the start of https://crate.io/docs/crate/reference/en/latest/run.html

also, for this intro para, I think there's no need to mention the name of the tool. just something along the lines of this document shows you how to manage nodes in your cluster